src/marni.cpp

```diff
@@ -2,6 +2,9 @@
 #include "interop.hpp"
 #include "re2.h"
 
+#include <ddraw.h>
+#include <windows.h>
```

Comment:
Above these headers, I think `#define _WIN32_LEAN_AND_MEAN` to reduce the bloat you get.